### PR TITLE
Skip PRs from the triage process

### LINKED ISSUE #12434: ampprojectbot: Don't automatically add PRs to milestones
Currently, PRs seem to be auto-added to the "Pending Triage" milestone. Then they get pinged regularly for not being triaged. We should constrain that behavior to `type:issue`.

Not sure if this is the right project/component. 

/to @adelinamart

### PR DESCRIPTION
**Pull request by adelinamart**
Fixes #12434
Added more batches also since the number of issues grew.

### CODE REVIEW

File: build-system/tasks/process-github-issues.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const BIWEEKLY_DAYS = 14;
 // days for quarterly updates
 const QUARTERLY_DAYS = 89;
 // we need around 10 batches to get 1k issues
```
Comment (rsimha-amp) [inline build-system/tasks/process-github-issues.js]: Update this comment.

File: build-system/tasks/process-github-issues.js [hunk 2]
```diff
@@ -136,6 +136,13 @@ function updateGitHubIssues() {
           const issueLastUpdate = issue.updated_at;
           let biweeklyUpdate = true;
           let quartelyUpdate = true;
+          // if an issue is an pull request will skip it
+          if (pullRequest) {
+            if (isDryrun) {
+              util.log(util.colors.red(issue.number + ' is pull request'));
```
Comment (rsimha-amp) [inline build-system/tasks/process-github-issues.js]: nit: "is **a** pull request"

File: build-system/tasks/process-github-issues.js [hunk 3]
```diff
@@ -136,6 +136,13 @@ function updateGitHubIssues() {
           const issueLastUpdate = issue.updated_at;
           let biweeklyUpdate = true;
           let quartelyUpdate = true;
+          // if an issue is an pull request will skip it
```
Comment (rsimha-amp) [inline build-system/tasks/process-github-issues.js]: if an issue is **a** pull request, **we'll** skip it

File: build-system/tasks/process-github-issues.js (hunk 1 above)
Comment (adelinamart) [reply build-system/tasks/process-github-issues.js]: done

File: build-system/tasks/process-github-issues.js (hunk 3 above)
Comment (adelinamart) [reply build-system/tasks/process-github-issues.js]: done

File: build-system/tasks/process-github-issues.js (hunk 2 above)
Comment (adelinamart) [reply build-system/tasks/process-github-issues.js]: done. Thank you!

**Comment by choumx** [comment]
Thanks for fixing this!

**Pull request merged**